pkg/reconciler/route/resources/ingress.go

```diff
@@ -339,6 +345,10 @@ func makeBaseIngressPath(ns string, targets traffic.RevisionTargets,
 		if t.LatestRevision != nil && *t.LatestRevision {
 			cfg = rolloutConfig(t.ConfigurationName, roCfgs)
 		}
+		servicePort := intstr.FromInt(networking.ServicePort(t.Protocol))
+		if internalTLS {
+			servicePort = intstr.FromInt(443)
```

Comment:
s/443/networking.ServiceHTTPSPort

Reply:
Yes, there are several reasons:
- non-admin users cannot notice if they are using secure traffic or not.
- It is easier to debug as you said.
- It makes easy to use Kingress by extra components. (OpenShift uses an extra controller to creates a LB resource based on the Kingress.)